tests/test_relations.py

```diff
@@ -145,9 +145,9 @@ def test_pk_representation(self):
         assert representation == self.instance.pk.int
 
 
-@override_settings(ROOT_URLCONF=[
+@override_settings(ROOT_URLCONF=(
     url(r'^example/(?P<name>.+)/$', lambda: None, name='example'),
-])
+))
```

Comment:
I'd drop this change as unrelated, but... 🤷‍♀ 

Out of scope but: Do these tests work as expected? Without looking right into the internals I'm not at all sure that `ROOT_URLCONF` really takes an iterable, rather than a string import path...?)

Reply:
Yeah I think you're right - it's not *at all* doing what it's supposed to be, and it's pure fluke that it's working.

I changes this fairly blindly, assuming it was a tuple vs. list issue, because without this change we get some test failures with `TypeError: unhashable type: 'list'` in Django's URL resolver. However I think the test cases are just flat-out-wrong here, and I've just resolved this into to a still broken, but not failing case.

Reply:
Grrr... bit of luck then hitting it. 😬

Maybe convert to a `URLPatternsTestCase` and see what really breaks?